### ISSUE TITLE
form-layout acceptance tests build error with ember-browserify

### ISSUE THREAD

**Issue by Truffula** [sir-dunxalot/ember-easy-form-extensions#32]
Using the 1.11.0.beta version with ember-browserify, the syntax used in `app/acceptance-tests/form-layout` is throwing build errors (in both the acorn and esprima versions of ember-browserify).

I'm not sure if it's invalid ES6 syntax or if it's just not recognised by acorn/esprima, but changing to the following fixes the issue:

``` js
import FormLayoutTest from 'ember-easy-form-extensions/acceptance-tests/form-layout';
export default FormLayoutTest;
```

**Comment by sir-dunxalot** [comment]
That is a really odd file. Not sure what it's doing there but I'll remove it with the next push to the branch.

**Comment by sir-dunxalot** [comment]
Fixed by #37
